css/global.css

```diff
@@ -1,31 +1,3 @@
-.jobs-unified-top-card__subtitle-secondary-grouping
-  > .jobs-unified-top-card__bullet {
-  display: none !important;
-}
-
-/* hides "over 100 applicants" message at linkedin.com/jobs/collections */
-
-.job-details-jobs-unified-top-card__primary-description-container:has(div):has(
-    .tvm__text--neutral
-  ):last-child {
-  display: none !important;
-}
-
-/* hides "see how you compare to x applicants" option next to reactivate premium button on view page */
-
-.job-details-jobs-unified-top-card__job-insight--highlight:has(div):has(
-    div
-  ):has(div):has(svg[data-test-icon="lightbulb-medium"]) {
-  display: none !important;
-}
-
-/* hides applicant count at top of page at linkedin.com/jobs/collections */
-.tvm__text--positive > strong {
-  display: none !important;
-}
-
-/* bold green text on linkedin.com/jobs/ next to the word promoted */
-
-.job-card-container__footer-item > strong {
+body {
```

Comment:
The simplest way to hide the reddit site was just using the body tag as the selector. Since the extension only actually works on `www.reddit.com`, there's no concern about potentially hiding the body tag on old.reddit.com as well.